### PR TITLE
Changelog 4.31.0

### PR DESCRIPTION
**Pull request by pmaytak**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,14 @@
+4.31.0
+==========
+### New Features:
+**Added time metrics to `AuthenticationResultMetadata`**, which includes total duration, time spent in HTTP, and duration of token cache callbacks. See pull request [#2581](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2581).  
+**Added telemetry data to requests sent to WAM**. See issue [#2562](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2562).  
+
+### Bug Fixes:
+**Fixed parenting of WAM account picker control**. See issue [#2566](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2566).  
+**Fixed an exception in console apps that use WAM when they are run directly as an executable**. See issue [#2608](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2608).  
+**Added option to hide iOS security prompt for system browser**. See issue [#2131](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2131).
```
Comment (jennyf19) [inline changelog.txt:10]: I think this is actually a feature request, and it brings a new API. @SameerK-MSFT might have thoughts on how to word it. Also, only impacts iOS 11+ (I think).

Comment (jennyf19) [reply changelog.txt:10]: also, @SameerK-MSFT  has the list of related issues, which we might want to include here as well. I think there are 3-4.

Comment (pmaytak) [reply changelog.txt:10]: Thanks for pointing that out, I'll move it to new features section.

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @pmaytak

**Pull request merged**